system-tests/test/deposit-redemption.test.ts

```diff
@@ -106,6 +126,21 @@ describe("System Test - Deposit and redemption", () => {
       signerOrProvider: depositor,
     })
 
+    walletRegistryHandle = new EthereumWalletRegistry({
+      address: walletRegistryAddress,
+      signerOrProvider: depositor,
+    })
+
+    const tbtcContracts: TBTCContracts = {
+      bridge: maintainerBridgeHandle,
+      tbtcToken: tbtcTokenHandle,
+      tbtcVault: vaultHandle,
+      walletRegistry: walletRegistryHandle,
+    }
+
+    walletTx = new WalletTx(tbtcContracts, electrumClient)
+    maintenanceService = new MaintenanceService(tbtcContracts, electrumClient)
```

Comment:
We should initialize SDK in the new fashion, see custom mode in: https://docs.threshold.network/app-development/tbtc-v2/tbtc-sdk/guides/initialize-sdk. All interactions should go through the `TBTC` entrypoint component and services (`sdk.deposits` etc.) or low-level contract access (`sdk.tbtcContracts`). If you need more signers, you can create an SDK instance for each one.

Ultimately, we should use standalone Ethers contract handles only for things that are not covered by the SDK.

Reply:
Done in 5691a18c4dff48f0fcfc198c32e245ed28903472.